app/presenters/collection_show_presenter.rb

```diff
@@ -1,6 +1,6 @@
 class CollectionShowPresenter < CurationConcerns::CollectionPresenter
   include CurationConcerns::Serializers
-  delegate :id, :title, :exhibit_id, :date_created, to: :solr_document, allow_nil: true
+  delegate :id, :title, :exhibit_id, :date_created, :keyword, to: :solr_document, allow_nil: true
```

Comment:
That seems reasonable, but we're not providing UI to edit them now — do we need to include keywords in the presenter, or can we just leave it out like we did when it was tags?